gradle/libs.versions.toml

```diff
@@ -0,0 +1,89 @@
+[versions]
+kotlin = "1.7.10"
+kotlin-coroutines = "1.6.4"
+java-target = "1.8"
```

Comment:
inline

Reply:
I inlined only coroutines, the other two are used in many many places.